### PR TITLE
Replace Queue With Subscription

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
Since changing to pub/sub, terminology is out of date so queue needs to become subscription

# What has changed
Changed queue to subscription

# Links
https://trello.com/c/XPwwUY31

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[Change exception manager to have more meaningful error fields (5)](https://trello.com/c/XPwwUY31/2662-change-exception-manager-to-have-more-meaningful-error-fields-5)

File: groundzero_ddl/casev3.sql [hunk 1]
```diff
@@ -127,7 +127,6 @@
        id uuid not null,
         destination_topic varchar(255),
         message_body bytea,
-        send_to_our_project boolean,
```
Comment (ryangrundy7) [inline groundzero_ddl/casev3.sql:130]: Don't think you meant to remove this?

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Don't think you meant to remove send_to_our_project

File: groundzero_ddl/casev3.sql (hunk 1 above)
Comment (NickGrantONS) [reply groundzero_ddl/casev3.sql:130]: it's ok... gone now anyway

**Review by NickGrantONS — APPROVED** [review]

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**